src/internal/pjsdb/queue_iterator.go

```diff
@@ -52,8 +52,17 @@ func NewQueuesIterator(extCtx sqlx.ExtContext, req IterateQueuesRequest) *Queues
 				array_agg(j.id ORDER BY j.id ASC) AS "jobs",
 				array_agg(j.program ORDER BY j.id ASC) AS "programs",
 				count(j.id) AS "size"
-			  FROM queues JOIN pjs.jobs j ON j.program_hash = queues.program_hash
-			  GROUP BY queues.program_hash`
+			  FROM 
+				queues 
+			  JOIN 
+				pjs.jobs j 
+				ON j.program_hash = queues.program_hash
+              WHERE
+                j.processing IS NULL
```

Comment:
Hey I don't think the queues iterator should automatically filter out jobs that are not done. Instead, that should be done in the filter. 

Rationale here is that you may want to see all jobs in a queue, even the done ones. There wouldn't be a way to do that through the API.

Reply:
Got it

Reply:
we discussed this in slack and decided we want a queue to be all jobs with the same program in a 'queued' stated only.